pages/business/setup/signup.tsx

```diff
@@ -88,6 +94,7 @@ const SetupSignupPage: React.FC<WithOwnerProps> = () => {
           email: '',
           password: '',
           confirmPassword: '',
+          confirmContract: false,
```

Comment:
```suggestion
          confirmContract: !isCareEnv && !isHealthEnv,
```
This should allow other users to register